futurex_openedx_extensions/helpers/tenants.py

```diff
@@ -314,3 +316,46 @@ def get_tenants_sites(tenant_ids: List[int]) -> List[str]:
 def get_nafath_sites() -> List:
     """Get all nafath sites"""
     return get_all_tenants_info()['special_info']['nafath_sites']
+
+
+def generate_tenant_config(name: str) -> dict:
+    """
+    Generate a tenant configuration by copying the default config and replacing placeholder PARAM.
+
+    :param name: Name to replace 'PARAM' in the default configuration
```

Comment:
please use `platform_name` instead of `name`. The more descriptive is the better
```suggestion
    :param platform_name: Name to replace `{{platform_name}}` in the default configuration
```

we also need `sub_domain` and `course_org_filter` as parameters

Reply:
As discussed we don't need course_org_filter separately as param. In default config to copy we can have somethings as follows which can be handled by sub_domain replacement.

```
"course_org_filter": [
    "{sub_domain}"
]
```